lib/ansible/modules/cloud/amazon/lightsail_lb.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: lightsail_lb
+short_description: Creates or deletes load balancer and attaches running instances to the created load balancer in AWS Lightsail.
+description:
+     - Create or delete a load balancer in AWS Lightsail.
+version_added: "2.8"
+author: "Gencebay Demir (@gencebay)"
+options:
+  state:
+    description:
+      - Indicate desired state of the target.
+    default: present
+    choices: ['present', 'absent']
+  name:
+    description:
+      - The name of the load balancer.
+    required: true
+  region:
```

Comment:
`region` is part of the standard ec2 documentation fragment and doesn't need to be re-documented here.

Reply:
Done